Open Judge System/Services/OJS.Services/Data/Settings/SettingsService.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using OJS.Common.Constants;
+using OJS.Data;
+using OJS.Data.Models;
+using OJS.Services.Cache;
+
+namespace OJS.Services.Data.Settings
+{
+    public class SettingsService : ISettingsService
+    {
+        private readonly IOjsDbContext dbContext;
+        private readonly ICacheService cacheService;
+
+        public SettingsService(IOjsDbContext dbContext, ICacheService cacheService)
+        {
+            this.dbContext = dbContext;
+            this.cacheService = cacheService;
+        }
+
+        public T Get<T>(string name, T type)
```

Comment:
Why you have parameter `T type`?

Reply:
The caller can specify in what type he wants to get the value. For example if the value is 200 , it can be returned as int.32 to the caller.

Reply:
Fixed